src/jdk.compiler/share/classes/com/sun/tools/javac/jvm/Gen.java

```diff
@@ -1977,8 +1969,8 @@ public void visitApply(JCMethodInvocation tree) {
         // the parameters of the method's external type (that is, any implicit
         // outer instance of a super(...) call appears as first parameter).
         MethodSymbol msym = (MethodSymbol)TreeInfo.symbol(tree.meth);
-        genArgs(tree.args,
-                msym.externalType(types).getParameterTypes());
+        List<Type> erasedPts = msym.externalType(types).getParameterTypes();
```

Comment:
Should we revert these lines to whatever is in mainline?

Reply:
sure will do